pkg/server/datastore/sqlstore/sqlstore.go

```diff
@@ -2662,12 +2666,12 @@ FROM
 WHERE registered_entry_id IN (SELECT id FROM listing)
 
 ORDER BY selector_id, dns_name_id
```

Comment:
We should also sort bu entry id here and for the MySQL variants.

Reply:
Good catch, done.